acceptance/acceptance_test.go

```diff
@@ -218,7 +218,11 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 
 			expanded := internal.ExpandEnvMatrix(config.EnvMatrix)
 
-			if len(expanded) == 1 && len(expanded[0]) == 0 {
+			if len(expanded) == 1 {
+				for _, envset := range expanded {
+					t.Logf("Running test with env %v", envset)
```

Comment:
since you asserted that `len(expanded) == 1` you can replace for loop with `t.Logf("Running test with env %v", expanded[0])`

I'd also add a comment `# Since env var is not part of the test log it`